### PR TITLE
[4.0] type added

### PR DESCRIPTION
**Pull request by puneeth-chanda**
Pull Request for Issue #23948

### Summary of Changes
'type="button" ' is added

### CODE REVIEW

**Comment by puneet0191** [comment]
have you made any changes in the flow of actions @puneeth2001 ? because the test keeps failing at the same point so doesn't seem like a false positive,maybe another eye from MM would help so @dneukirchen please help with this.

**Comment by dneukirchen** [comment]
update with latest dev and resolving conflicts should fix the tests. This is an expected failure. 

see:
https://github.com/joomla/joomla-cms/pull/23756
https://github.com/joomla/test-system/pull/53

File: administrator/components/com_media/layouts/toolbar/delete.php [hunk 1]
```diff
@@ -13,6 +13,6 @@
 
 $title = Text::_('JTOOLBAR_DELETE');
 ?>
-<button id="mediaDelete" class="btn btn-sm btn-danger" onclick="MediaManager.Event.fire('onClickDelete');">
-    <span class="icon-delete" title="<?php echo $title; ?>"></span> <?php echo $title; ?>
+<button id="mediaDelete" class="btn btn-sm btn-danger" onclick="MediaManager.Event.fire('onClickDelete');" type="button">
+        <span class="icon-delete" title="<?php echo $title; ?>"></span> <?php echo $title; ?>
```
Comment (Quy) [inline administrator/components/com_media/layouts/toolbar/delete.php]:
```suggestion
	<span class="icon-delete" title="<?php echo $title; ?>"></span> <?php echo $title; ?>
```

**Comment by brianteeman** [comment]
Can you look at resolving the conflicts so that this can be tested please.

**Comment by Quy** [comment]
This is no longer applicable.
```
<joomla-toolbar-button class="btn btn-sm btn-info" onclick="MediaManager.Event.fire('onClickCreateFolder');">
	<span class="icon-folder-close"></span> Create New Folder</joomla-toolbar-button>
```
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24033">issues.joomla.org/tracker/joomla-cms/24033</a>.</sub>

**Pull request closed without merging**